.github/workflows/lint_and_test_chart.yml

```diff
@@ -1,7 +1,10 @@
 name: Lint and Test Charts
 
-on: push
-
+on: 
+  push:
+  pull_request:
+        types:
+            - opened
```

Comment:
I think we need it also on synchronized and reopened. Otherwise you won't trigger the linting, after you correct some bugs / errors.

Reply:
I believe its  a good suggestion to have also have synchronized and reopened.